plugins/brimcap/brimcap-plugin.ts

```diff
@@ -214,11 +217,78 @@ export default class BrimcapPlugin {
     )
   }
 
+  private setupLoader() {
+    const load = async (
+      params: IngestParams & {spaceId: string},
+      onProgressUpdate: (value: number | null) => void,
+      onWarning: (warning: string) => void,
+      onDetailUpdate: () => void
+    ): Promise<void> => {
+      const {fileListData, name} = params
+      if (fileListData.length > 1)
+        throw new Error("Only one pcap can be opened at a time.")
```

Comment:
Is this still true with brimcap?

Reply:
looks to be the case: https://github.com/brimdata/brimcap/blob/f95d829bfd0560f4aa53590ee30380ac123520c1/cmd/brimcap/load/command.go#L97